src/AppBundle/Entity/Club.php

```diff
@@ -12,11 +15,7 @@
  */
 class Club
 {
-    public function __construct()
-    {
-        $this->created_at = new \DateTime('now');
-        $this->users = new ArrayCollection();
-    }
+    use TimestampableTrait;
```

Comment:
please don't use traits

Reply:
Why ?

Reply:
Please check it:
https://symfony.com/doc/current/doctrine/common_extensions.html
and
https://github.com/Atlantic18/DoctrineExtensions/blob/master/doc/timestampable.md

I do not thinks is the better way but it's very comfortable.

Reply:
https://dev.to/barryosull/why-i-dont-like-traits We just need createdAt field, just put in constructor $this->createdAt = new DateTime();

Reply:
Yes it's not a best way, but useful. When i add extra fields for dates like addedTime etc. i know that fields are not participate in some programming patterns. I know the projects with that solution and some different opinion not eliminate this idea. But I know the disadvantages of this solution.